### DISCUSSION TITLE
Disable timestamps for whisper?

### DISCUSSION THREAD

**Discussion by Rrrila** [openai/whisper#1333] [Q&A] [answered]
Hi,

I have been searching all over the internet, including the official documentation of whisper, but i cant find a way to disable timestamps on whisper transcripts. I'm using a colab.land project with the following line:

`!whisper {input_path} --model large-v2 --language English --output_dir {output_folder} --output_format vtt`

Can you help me on this? I’m not a developer myself, so I might have miss something. I have seen other hugging face projects where you can actually choose activate or deactivate timestamps for the output.

Thanks, Kind regards

**Comment by ryanheise** [comment, marked as answer]
The `vtt` format is intended for timed subtitles. I think for a non-developer, the closest option to what you want is `--output_format json`. Then when you open up that JSON file in a text editor, you'll see the plain text very close to the top of the file:

```javascript
{"text": " THE FULL TEXT WILL APPEAR HERE", ...
```

You can then copy and paste that part of the file.

If you want a command to extract the text out of the JSON file, you can also try the following (after you've already run Whisper to output `yourfile.json`):

```
!cut -d'"' -f4 yourfile.json
```

This cuts up the file into the different parts between and around the `"` characters and prints out the 4th part which (at least currently) happens to be the text that you want to grab.

**Reply by Rrrila** [reply]
That is what i was thinking about actually.. but isnt there txt format? wouldnt that be plain text?

**Comment by chechotka88** [comment]
Hi, I had the same issue and accidentally found a solution, hope it will work for you as well.
Run the same code, just add a command to choose an output directory where a file with selected output format will be created. Choose a txt, it will not include timestamps.
`!whisper {input_path} --model large-v2 --language English --output_dir {output_folder} --output_format txt --output_dir {output_path}`
